<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>H2Panel</class>
 <widget class="QWidget" name="H2Panel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="2" column="1">
       <widget class="QLabel" name="label_7">
        <property name="maximumSize">
         <size>
          <width>1677516</width>
          <height>1677516</height>
         </size>
        </property>
        <property name="text">
         <string>Tick(ms)</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QToolButton" name="btnMarkNow">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Smart Add</string>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/res/image/icon/addition.png</normaloff>:/res/image/icon/addition.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="spinBox">
        <property name="minimum">
         <number>100</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="chkOnOff">
        <property name="text">
         <string>ON/OFF</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4" colspan="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>X:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MegaLcdnumber" name="lcdX">
          <property name="digitCount">
           <number>9</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Y:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MegaLcdnumber" name="lcdY">
          <property name="digitCount">
           <number>9</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0" colspan="2">
       <widget class="PageOption" name="widget" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string/>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <property name="horizontalSpacing">
          <number>3</number>
         </property>
         <property name="verticalSpacing">
          <number>3</number>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>t(s)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="spinT">
           <property name="minimum">
            <double>0.010000000000000</double>
           </property>
           <property name="maximum">
            <double>100.000000000000000</double>
           </property>
           <property name="value">
            <double>3.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>X</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="spinX">
           <property name="minimum">
            <double>-1000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1000.000000000000000</double>
           </property>
           <property name="value">
            <double>10.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Y</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="spinY">
           <property name="minimum">
            <double>-1000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1000.000000000000000</double>
           </property>
           <property name="value">
            <double>10.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QToolButton" name="btnMarkUser">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Smart Add</string>
           </property>
           <property name="text">
            <string>Mark</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/res/image/icon/addition.png</normaloff>:/res/image/icon/addition.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QToolButton" name="btnGo">
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Go</string>
           </property>
           <property name="text">
            <string>GO</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/res/image/icon/skip.png</normaloff>:/res/image/icon/skip.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="chkSingle">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Single</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QGroupBox" name="groupBox_6">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <property name="leftMargin">
             <number>3</number>
            </property>
            <property name="topMargin">
             <number>3</number>
            </property>
            <property name="rightMargin">
             <number>3</number>
            </property>
            <property name="bottomMargin">
             <number>3</number>
            </property>
            <property name="spacing">
             <number>3</number>
            </property>
            <item row="1" column="1">
             <widget class="QToolButton" name="btnCenter">
              <property name="maximumSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/res/image/icon3/address.png</normaloff>:/res/image/icon3/address.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QToolButton" name="btnYN">
              <property name="maximumSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Step Y</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/res/image/icon/c_back.png</normaloff>:/res/image/icon/c_back.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="shortcut">
               <string>Ctrl+Left</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QToolButton" name="btnXN">
              <property name="maximumSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Step X</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/res/image/icon/c_subtract.png</normaloff>:/res/image/icon/c_subtract.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="shortcut">
               <string>Ctrl+Down</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QToolButton" name="btnXP">
              <property name="maximumSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Step X</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/res/image/icon/c_add.png</normaloff>:/res/image/icon/c_add.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="shortcut">
               <string>Ctrl+Up</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QToolButton" name="btnYP">
              <property name="maximumSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Step Y</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/res/image/icon/c_more.png</normaloff>:/res/image/icon/c_more.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="shortcut">
               <string>Ctrl+Right</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3" rowspan="3">
             <widget class="QLabel" name="label_8">
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>120</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>:/res/image/joint/mrx-h2_geo.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="4">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string>Step</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="leftMargin">
             <number>3</number>
            </property>
            <property name="topMargin">
             <number>6</number>
            </property>
            <property name="rightMargin">
             <number>3</number>
            </property>
            <property name="bottomMargin">
             <number>3</number>
            </property>
            <property name="spacing">
             <number>3</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>t(s)</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="spinStepT">
              <property name="minimum">
               <double>0.010000000000000</double>
              </property>
              <property name="maximum">
               <double>100.000000000000000</double>
              </property>
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Step(mm)</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="spinStepV">
              <property name="minimum">
               <double>0.010000000000000</double>
              </property>
              <property name="maximum">
               <double>1000.000000000000000</double>
              </property>
              <property name="value">
               <double>20.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MegaLcdnumber</class>
   <extends>QLCDNumber</extends>
   <header>megalcdnumber.h</header>
  </customwidget>
  <customwidget>
   <class>PageOption</class>
   <extends>QWidget</extends>
   <header>pageoption.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
